### ISSUE TITLE
bug: SNS publish to subscribed SQS queue Message Attributes not valid serialization

### ISSUE THREAD

**Issue by ivanmartyniuk9** [localstack/localstack#6549]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi, I am using C# and AWSSDK.SimpleNotificationService, and AWSSDK.SQS libraries.I created in localstack topic subscribed existsing queue to it.Then in my code I publish message to topic with some message attributes has format 
```
 "MessageAttributes": {
                "key": {
                    "StringValue": "blalbla",
                    "DataType": "String"
                }
            }
```
![image](https://user-images.githubusercontent.com/86060333/181771956-1d50d789-ef4d-4211-b22e-3b943a91d9ce.png)

### Expected Behavior

` "MessageAttributes": {
                "key": {
                    "Value": "blalbla",
                    "Type": "String"
                }
            }`
            
![image](https://user-images.githubusercontent.com/86060333/181772296-f81fe712-ed58-4cce-b216-8b6898e71b7f.png)

### How are you starting LocalStack?

With a `docker run` command

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker run localstack/localstack

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)
aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name test-queue
aws --endpoint-url=http://localhost:4566 sns create-topic --name test-topic
aws --endpoint-url=http://localhost:4566 sns subscribe --topic-arn arn:aws:sns:us-east-1:000000000000: test-topic --protocol sqs --notification-endpoint http://localhost:4566/000000000000/test-queue

publish message from C# libraries to topic with MessageAttributes

### Environment

```markdown
- OS: Windows 10
- LocalStack:latest
```

**Comment by bentsku** [comment]
Hello @ivanmartyniuk9,

This should be fixed with #6373. Can you try to pull the latest Docker image with docker pull localstack/localstack:latest and give it a try? Your issue should be resolved.

**Comment by bentsku** [comment]
Thank you, I'm glad it's working! Don't hesitate if there's anything else. Have a good day!
